x/gov/client/cli/query.go

```diff
@@ -43,7 +43,7 @@ $ gaiacli query gov proposal 1
 
 			var proposal gov.Proposal
 			cdc.MustUnmarshalJSON(res, &proposal)
-			return cliCtx.PrintOutput(proposal)
+			return cliCtx.PrintOutput(proposal) // nolint:errcheck
```

Comment:
why do we need this `errcheck` suppression for lines like these? Is there anything we can do to fix this lint issue? what's causing it?

Reply:
`PrintOutput()` returns err

Reply:
Looks like a bug in `errcheck` to me...

Reply:
if 
```
if err := cliCtx.PrintOutput(proposal); err != nil {
return err
} 
return nil 
```
is not an error but what we have _is_ an error, then yes, it looks like an `errcheck` bug to me